pkgs/tools/typesetting/tex/texlive/combine.nix

```diff
@@ -38,16 +38,34 @@ let
   mkUniqueOutPaths = pkgs: uniqueStrings
     (map (p: p.outPath) (builtins.filter lib.isDerivation pkgs));
 
-in (buildEnv {
   name = "texlive-${extraName}-${bin.texliveYear}${extraVersion}";
 
-  extraPrefix = "/share/texmf";
+  texmf = (buildEnv {
+    name = "${name}-tex";
+    extraPrefix = "/share/texmf";
+    ignoreCollisions = false;
+    paths = pkgList.nonbin;
+    postBuild =
+    # link info and man pages
+    ''
+      for d in {info,man}; do
+        [[ -e "$out/share/texmf/doc/$d" ]] && ln -s "texmf/doc/$d" "$out/share/$d"
+      done
+    '';
```

Comment:
Can we replace this with ``outputsToInstall``?

Reply:
If I understand correctly (both the question and `texlive.combine`), not in the way the packages are generated. For this `buildEnv`, each package is bringing only one output.